recipes/clapack/build.sh

```diff
@@ -0,0 +1,13 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
```

Comment:
Please try `CPPFLAGS="-I${PREFIX}/include" $CPPFLAGS` in all of these variables to preserve the original settings.